config/routes.rb

```diff
@@ -127,7 +127,6 @@
       get '/login/two_factor/sms/:opt_out_uuid/opt_in' => 'two_factor_authentication/sms_opt_in#new',
           as: :login_two_factor_sms_opt_in
       post '/login/two_factor/sms/:opt_out_uuid/opt_in' => 'two_factor_authentication/sms_opt_in#create'
-      get '/login/two_factor/otp_expired' => 'two_factor_authentication/otp_expired#show'
```

Comment:
I'm assuming the feature is disabled so we don't need to worry about backwards compatibility?

Reply:
> I'm assuming the feature is disabled so we don't need to worry about backwards compatibility?

Good call-out, but yes, the feature has been disabled for quite a whiel.